fluent-dom/src/localization.js

```diff
@@ -82,12 +99,10 @@ export default class Localization {
   /**
    * Retrieve translations corresponding to the passed keys.
    *
-   * A generalized version of `DOMLocalization.formatValue`. Keys can
-   * either be simple string identifiers or `[id, args]` arrays.
-   *
    *     docL10n.formatValues([
    *       ['hello', { who: 'Mary' }],
-   *       ['hello', { who: 'John' }]
+   *       ['hello', { who: 'John' }],
+   *       ['welcome']
```

Comment:
This could be written simply as `'welcome'` to illustrate the point about passing simple strings.